package.json

```diff
@@ -97,6 +97,7 @@
     "@ministryofjustice/frontend": "^2.1.3",
     "agentkeepalive": "^4.5.0",
     "applicationinsights": "^2.9.5",
+    "axe-core": "^4.7.2",
```

Comment:
any reason for this to be a standard dependency instead of a dev dependency?

Reply:
No, moved to dev dependencies